corral/bundle/json.pony

```diff
@@ -1,44 +1,44 @@
 use "files"
 use "json"
-use "../util"
+use "logger"
 
 primitive JsonError
 
 primitive Json
-  fun load_object(file_path: FilePath, log: Log)
+  fun load_object(file_path: FilePath, log: Logger[String])
     : (JsonObject | FileErrNo | JsonError)
   =>
-    //log.fine("Reading json from: " + file_path.path)
+    //log(Fine) and log.log("Reading json from: " + file_path.path)
     let file = match OpenFile(file_path)
     | let f: File => f
     | let e: FileErrNo => return e
     end
     let content: String = file.read_string(file.size())
-    //log.fine("Read: " + content + ".")
+    //log(Fine) and log.log("Read: " + content + ".")
```

Comment:
i'd say lets kill these "debug" statements that are commented out. in another pr.

Reply:
Ok, I'll open a separate PR after this one is merged.